package.json

```diff
@@ -58,8 +58,8 @@
     "node-fetch": "2.6.13"
   },
   "engines": {
-    "node": "16.20.2",
-    "npm": "8.19.4"
+    "node": "^20",
```

Comment:
We don't really need to pin a version of node/npm here, else as soon as a new minor/patch comes out warnings will show which is annoying.